rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1044,66 +1084,65 @@ private void processReleaseRequests() {
             // Further logging is done at the tx builder level.
             if (!result.isPresent()) {
                 logger.warn(
-                        "Couldn't build a release BTC tx for <{}, {}>",
-                        releaseRequest.getDestination().toBase58(),
-                        releaseRequest.getAmount());
+                    "Couldn't build a release BTC tx for <{}, {}>",
+                    releaseRequest.getDestination().toBase58(),
+                    releaseRequest.getAmount());
                 return false;
```

Comment:
```suggestion
                return;
```
It's a void method really

Reply:
The Return Is For The Lamdba in releaseRequestQueue.process() method

Reply:
oh right. thanks